ticket-app/src/main/java/com/chudzik/enums/TicketType.java

```diff
@@ -0,0 +1,19 @@
+package com.chudzik.enums;
+
+import java.math.BigDecimal;
+
+public enum TicketType {
+    ADULT(BigDecimal.valueOf(25)),
+    STUDENT(BigDecimal.valueOf(18)),
+    CHILD(BigDecimal.valueOf(12.5));
```

Comment:
Gdzie moglibyśmy umieścić te kwoty by w razie podwyżek nie musieć budować aplikacji od nowa?

Reply:
Możnaby je ustawić jako environment variable. Dodatkowo eksperymentowałem podczas developmentu z traktowaniem TicketType jako encji z własnym repozytorium, ale po pewnym czasie pozostawiłem to jako enum.